core/store/models/external_initiator.go

```diff
@@ -19,14 +19,16 @@ type ExternalInitiatorRequest struct {
 
 // ExternalInitiator represents a user that can initiate runs remotely
 type ExternalInitiator struct {
-	*gorm.Model
```

Comment:
Should we also remove the underlying SQL columns in a migration, and perhaps convert the PK from `ID` to `Name`? The latter might be frustrating with SQLite.


`gorm.Model`
```
type Model struct {
	ID        uint `gorm:"primary_key"`
	CreatedAt time.Time
	UpdatedAt time.Time
	DeletedAt *time.Time `sql:"index"`
}
```

Reply:
Sounds good to me :+1:, I'm always a little concerned about the migrations leading to downtime.  Should this be a concern?

Reply:
Fair point. Let's chat about it at standup.

Reply:
@dimroc it looks like sqlite3 [doesn't support seamlessly changing the primary key](https://stackoverflow.com/questions/16900552/change-the-primary-key-of-a-table-in-sqlite).